src/concrete_difference.jl

```diff
@@ -0,0 +1,69 @@
+export difference
+
+# alias for set difference
+import Base: \
+
+"""
+    \\(X::LazySet, Y::LazySet)
+
+Convenience alias for set difference.
+
+### Input
+
+- `X` -- a set
+- `Y` -- another set
+
+### Output
+
+The set difference between `X` and `Y`.
+"""
+\(X::LazySet, Y::LazySet) = difference(X, Y)
+
+# =================================
+# Set difference between intervals
+# =================================
+
+"""
+    difference(I1::Interval{N}, I2::Interval{N}) where {N}
+
+Return the set difference between the given intervals.
+
+The set difference is defined as:
+
+```math
+    I1 \\ I2 = \\{x: x ∈ I₁~\\textrm{and} x ∉ I₂ \\}
```

Comment:
```suggestion
    I1 \\setminus I2 = \\{x: x ∈ I₁ \\text{ and } x ∉ I₂ \\}
```
I am not sure that `\\` displays `\`. I suspect that you need `\\\\` in LaTeX. But `\\setminus` is the better way.

Reply:
I haven't checked how does it display (cannot build docs locally, i hit the `Expokit` dependency problem).

Note that the alias that i used is the usual backslash, `\`, not `\setminus[TAB]`. I tried to use `setminus` at first but julia doesn't like it; it says `ERROR: syntax: invalid character "∖" `.

Reply:
It is fine to not use `\setminus` in the code, but I prefer it in formulas in the docs.